docs/user-guide/install.md

```diff
@@ -24,19 +24,19 @@ VAAPI hardware-accelerated transcoding: `jasongdove/ersatztv:develop-vaapi`
 
 ### Docker
 
-1. Download the latest container image
```

Comment:
These are the line breaks that are breaking the ordered list. By escaping `.` for steps 2 and 3, the step numbers render properly.